rcl/src/rcl/remap_impl.h

```diff
@@ -51,10 +51,6 @@ typedef struct rcl_remap_impl_t
   rcl_allocator_t allocator;
 } rcl_remap_impl_t;
 
-/// Get an rcl_remap_t structure initialized with NULL.
-rcl_remap_t
-rcl_remap_get_zero_initialized();
```

Comment:
this one don't even have a definition, right?

Reply:
Yes, this probably is dead code targeting [`rcl_get_zero_initialized_remap`](https://github.com/ros2/rcl/blob/3bd0871658826a253aae358fe8569d1a057c7cc5/rcl/include/rcl/remap.h#L42)